test/Model.spec.js

```diff
@@ -496,4 +496,30 @@ describe("Model", () => {
             });
         });
     });
+
+    describe("set and get", () => {
+        it("should throw an error on invalid input when calling Model:get", (done) => {
+            const model = new Model();
+            model.get("{\"foobar\":[]}").subscribe(() => {
+                done(new Error("Did not receive an error when one was expected"));
```

Comment:
This should be moved to the onComplete handler, rather than onNext. (With rx Observable, onComplete and onError are mutually exclusive: either terminates the sequence.)